api/reporting/docs/cohort_definition_schema.json

```diff
@@ -0,0 +1,201 @@
+          ]
+        },
+        "temporal": {
+          "description": "Is the search group temporal(relating groups within a time frame). ",
+          "type": "boolean"
+        }
+      },
+      "required": ["id", "items", "temporal"]
+    },
+    "search_group_item": {
+      "description": "A search group item is a collection of concepts of the same domain which can be 'OR'ed together",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The unique identifier for the search group item",
+          "type": "string"
+        },
+        "type": {
+          "description": "The description for this search group item",
+          "type": "string"
+        },
+        "searchParameters": {
+          "description": "A collection search parameters",
+          "type": "array",
+          "items": [
+            {
+              "$ref": "#/definitions/search_parameter"
+            }
+          ]
+        },
+        "modifiers": {
+          "description": "A collection of modifiers that refine the search",
+          "type": "array",
+          "items": [
+            {
+              "$ref": "#/definitions/modifier"
+            }
+          ]
+        },
+        "temporalGroup": {
+          "description": "Temporal group index for the search group item",
+          "type": "integer"
+        }
+      },
+      "required": ["id", "type", "searchParameters", "modifiers"]
+    },
+    "search_parameter": {
+      "description": "A search parameter contains metadata about a specific concept",
+      "type": "object",
+      "properties": {
+        "parameterId": {
+          "description": "The unique identifier for the search parameter",
+          "type": "string"
+        },
+        "name": {
+          "description": "The concept name",
+          "type": "string"
+        },
+        "domain": {
+          "description": "The domain for this concept",
+          "type": "string"
+        },
+        "type": {
+          "description": "The vocabulary type",
+          "type": "string"
+        },
+        "group": {
+          "description": "Is the concept a parent or child",
+          "type": "boolean"
+        },
+        "attributes": {
+          "description": "A collection of attributes that can be searched for specific domains, ex: labs",
+          "type": "array",
+          "items": [
+            {
+              "$ref": "#/definitions/attribute"
+            }
+          ]
+        },
+        "ancestorData": {
+          "description": "Does the search parameter contain data in the ancestor table",
+          "type": "boolean"
+        },
+        "standard": {
+          "description": "Is the search parameter source or standard",
+          "type": "boolean"
+        },
+        "conceptId": {
+          "description": "The concept id",
+          "type": "integer"
+        },
+        "value": {
+          "description": "For PPI questions this holds the answer concept id",
```

Comment:
So this is a non-required  property, and will be absent if not applicable?

Reply:
Turns out this is deprecated and no longer used in the codebase, but we still have json cohort definitions that include it. So added deprecation to description and created a ticket to remove this property from the SearchParameter: https://precisionmedicineinitiative.atlassian.net/browse/RW-5680